src/components/product-grid/product-grid.js

```diff
@@ -10,7 +10,14 @@ function init() {
   componentEl = document.querySelector('[data-component="grid"]');
 }
 function update(products){
-  componentEl.innerHTML = renderTiles({products});
+  if(products){
+    if(componentEl.style.display = 'block')
```

Comment:
should be `==` or `===` here.

I'm also wondering if the check is necessary. What's the harm in just always setting the value to `grid` in this case?

Reply:
@aevanson - if the display is grid, the error message is squished into a block that is 1/3 of the width.

Reply:
I have updated